BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonRequest.java

```diff
@@ -64,19 +68,20 @@ public BaragonRequest(@JsonProperty("loadBalancerRequestId") String loadBalancer
     this.replaceUpstreams = Objects.firstNonNull(replaceUpstreams, Collections.<UpstreamInfo>emptyList());
     this.noValidate = Objects.firstNonNull(noValidate, false);
     this.noReload = noReload;
+    this.noServiceUpdate = noServiceUpdate;
```

Comment:
i think there's a situation where `noServiceUpdate` and `replaceServiceId` being used together can screw things up